lib/adagrams.rb

```diff
@@ -0,0 +1,153 @@
+require 'csv'
+
+def draw_letters
+  all_letters_hash = {
+    "A" => 9,
+    "B" => 2,
+    "C" => 2,
+    "D" => 4,
+    "E" => 12,
+    "F" => 2,
+    "G" => 3,
+    "H" => 2,
+    "I" => 9,
+    "J" => 1,
+    "K" => 1,
+    "L" => 4,
+    "M" => 2,
+    "N" => 6,
+    "O" => 8,
+    "P" => 2,
+    "Q" => 1,
+    "R" => 6,
+    "S" => 4,
+    "T" => 6,
+    "U" => 4,
+    "V" => 2,
+    "W" => 2,
+    "X" => 1,
+    "Y" => 2,
+    "Z" => 1,
+  }
+  all_letters_array = Array.new
+  all_letters_hash.each do |letter, number|
+    number.times do
+      all_letters_array << letter
+    end
+  end
+  letters_in_hand = all_letters_array.sample(10)
```

Comment:
You should explicitly return `letters_in_hand`. Note warning when you run rake:

"/Users/becca/Documents/GitHub/c12/adagrams/lib/adagrams.rb:38: warning: assigned but unused variable - letters_in_hand"